Update azure-stack-enable-multitenancy.md

rtibi opened the pull request:
You will also need to add the AzureStackAdmin environment as if not added before 
+      ````PowerShell
+        Add-AzureRmEnvironment -Name "AzureStackAdmin" -ArmEndpoint "https://adminmanagement.local.azurestack.external"
     ```` 
+    > if you don't, running the first part of the onboarding fails with the following message:
+  "
+    Add-AzureRmAccount : Value cannot be null.
+    Parameter name: uriString
+    At C:\AzureStack\AzureStack-Tools-vnext\Identity\AzureStack.Identity.psm1:200 char:25
+  "

azurecla commented:
Hi __@roopalik__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rtibi committed:
Update azure-stack-enable-multitenancy.md

You will also need to add the AzureStackAdmin environment as if not added before
